scanner/image/db/Dockerfile

```diff
@@ -26,19 +28,20 @@ COPY signatures/RPM-GPG-KEY-PGDG-13 /
 COPY scripts/docker-entrypoint.sh scripts/init-entrypoint.sh /usr/local/bin/
 COPY --from=postgres_rpms /rpms/postgres.rpm /rpms/postgres-libs.rpm /rpms/postgres-server.rpm /rpms/postgres-contrib.rpm /tmp/
 
-RUN microdnf upgrade --nobest && \
+RUN microdnf upgrade -y --nobest && \
     # groupadd is in shadow-utils package that is not installed by default.
-    microdnf install shadow-utils && \
+    microdnf install -y shadow-utils && \
     groupadd -g 70 postgres && \
     adduser postgres -u 70 -g 70 -d /var/lib/postgresql -s /bin/sh && \
-    microdnf install \
+    microdnf install -y \
         ca-certificates libicu systemd-sysv \
         glibc-locale-source glibc-langpack-en \
         perl-libs libxslt && \
+    if [[ "$(uname -m)" == "s390x" ]]; then microdnf install -y python3 uuid; fi && \
```

Comment:
This is actually related to using ubi 9 minimal instead of it running on s390x, but we only use ubi 9 minimal for s390x, and this was a simple check

Reply:
Similar changes in https://github.com/stackrox/stackrox/pull/7504.  Should we guard those as well?
Perhaps we should put a comment in code or guard using release (a bit verbose, but it works):
```
if [ $(awk -F'=' '/VERSION_ID/{ gsub(/"/,""); print substr($2,1,1)}' /etc/os-release) -gt 7 ] ; then
   ...
fi
```